scripts/run_mmdeply.sh

```diff
@@ -0,0 +1,19 @@
+mkdir ./weight/mmdet_faster_rcnn/
```

Comment:
What about split it into 2 scripts, `deploy_mmdet_faster_rcnn` and `deploy_mmpose_hrnet`.